src/dos/dos.cpp

```diff
@@ -1204,6 +1204,25 @@ static Bitu DOS_26Handler(void) {
     return CBRET_NONE;
 }
 
+void set_ver(char *args, bool start)
+{
+	char *word = StripWord(args);
+	if (!*args && !*word) { // Reset
+		dos.version.major = 5;
+		dos.version.minor = 0;
+	} else if (*args == 0 && *word && (strchr(word, '.') != 0)) { // Allow:
```

Comment:
Suggest an assert at the top to check if either `args` or `word` are NULL.

Reply:
No problem!